crates/typst-render/src/lib.rs

```diff
@@ -387,6 +392,44 @@ fn render_outline_glyph(
         let scale = text.size.to_f32() / text.font.units_per_em() as f32;
 
         let mut pixmap = None;
+
+        let rule = sk::FillRule::default();
+
+        // Flip vertically because font design coordinate
+        // system is Y-up.
+        let ts = ts.pre_scale(scale, -scale);
+        if let Some(FixedStroke {
+            paint,
+            thickness,
+            line_cap,
+            line_join,
+            dash_pattern,
+            miter_limit,
+        }) = &text.stroke
+        {
+            if thickness.to_f32() > 0.0 {
+                let dash = dash_pattern.as_ref().and_then(to_sk_dash_pattern);
+
+                let paint = to_sk_paint(
+                    paint,
+                    state.pre_concat(sk::Transform::from_scale(scale, -scale)),
+                    Size::zero(),
+                    false,
+                    None,
+                    &mut pixmap,
+                    None,
+                );
+                let stroke = sk::Stroke {
+                    width: thickness.to_f32() / scale, // When we scale the path, we need to scale the stroke width, too.
```

Comment:
short answer: I don't know. Adding this make the result looks correct.

Long answer:
This might because, each glyph has an extra scale, and it is defined by
```
let scale = text.size.to_f32() / text.font.units_per_em() as f32;
```
This is used to scale glyph paths stored in font file and it doesn't exists in shapes. Therefore the width of our stokes are wrongly scaled so we need to cancel it by diving it.